mac/INSTALL.md

```diff
@@ -1,18 +1,27 @@
 Installation
 ============
 
-- Install Python3
+- Install **Python3**
 
-- Install homebrew:
-  /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+- Install **XCode Command Line Tools**: `xcode-select --install`
```

Comment:
Why is XCode required? We're not building anything.

Reply:
Installing Homebrew requires Xcode CLT. And I think installing Python**3** on macOS does too.